py/desisim/scripts/quickquasars.py

```diff
@@ -200,7 +201,10 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
         log.warning("No TARGETID")
         targetid=None
 
-    sim_spectra(qso_wave,qso_flux, args.program, obsconditions=obsconditions,spectra_filename=ofilename,seed=args.seed,sourcetype="qso", skyerr=args.skyerr,ra=metadata["RA"],dec=metadata["DEC"],targetid=targetid)
+    log.warning("Assuming the healpix scheme is 'NESTED'")
```

Comment:
I don't think we need this warning.  By definition DESI uses the nested scheme everywhere; it isn't an option that requires an assumption about which one to use.